src/NLog/Config/XmlLoggingConfiguration.cs

```diff
@@ -65,10 +65,14 @@ public class XmlLoggingConfiguration : LoggingConfiguration
     {
         #region private fields
 
-        private readonly ConfigurationItemFactory configurationItemFactory = ConfigurationItemFactory.Default;
         private readonly Dictionary<string, bool> visitedFile = new Dictionary<string, bool>(StringComparer.OrdinalIgnoreCase);
 
         private string originalFileName;
+        
+        private ConfigurationItemFactory ConfigurationItemFactory
```

Comment:
This is connected to the other change you queried. It facilitates the lazy instantiation of the `ConfigurationItemFactory`.